service/job/job_controller.py

```diff
@@ -40,9 +42,16 @@ def job_create(job_type):
     :return: A JSON object containing the status, the job id and the task ids
         of every subtask in the chain
     """
-    params = request.get_json(force=True, silent=True)
```

Comment:
ist das im Endeffekt nicht genau das was silent=True macht ?

Reply:
Nein, bei `silent=True` wird auch syntaktisch nicht korrektes JSON einfach ignoriert.

Reply:
dann vieleicht try and raise ApiError für params = request.get_json(force=True) ?

Reply:
Das Problem ist, dass `request.get_json()` keinen Fehler wirft, sondern einen Callback aufruft (siehe http://flask.pocoo.org/docs/1.0/api/#flask.Request.get_json). Ich habe es jetzt so gelöst, dass der Error, der vom Standard-Callback geworfen wird in `run_service.handle_http_exception` abgefangen wird.